api/tests/patientDelete.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+curl localhost:3333
+echo
+source login.sh
+echo -en '\nShould Succeed\n'
+curlHeaders localhost:3333/patient/7 -X DELETE
```

Comment:
It would be better to have a test that first creates a patient, then deletes it, so that you can run the test multiple times.